lib/cli/npm.js

```diff
@@ -54,6 +55,38 @@ export default class NPM {
     })).json.latest;
   }
 
+  async getLatestSafeUpgradeVersion (pkg, curVersion) {
+    const allVersions = (await this.exec('view', [pkg, 'versions'], {
+      json: true
+    })).json;
+    return this.getLatestSafeUpgradeFromVersions (curVersion, allVersions);
+  }
+
+  getLatestSafeUpgradeFromVersions (curVersion, allVersions) {
```

Comment:
a docstring with parameter and return types description would be a good addition to this method